test/integrationTests/launchConfiguration.integration.test.ts

```diff
@@ -18,6 +19,12 @@ chai.use(require('chai-fs'));
 suite(`Tasks generation: ${testAssetWorkspace.description}`, function () {
     suiteSetup(async function () {
         should();
+
+        // These tests don't run on the BasicRazorApp2_1 solution
+        if (vscode.workspace.workspaceFolders[0].uri.fsPath.split(path.sep).pop() === 'BasicRazorApp2_1') {
```

Comment:
Maybe extract this out into a helper function `isRazorWorkspace(vscode.workspace)`